test/unit/sql/insert.test.js

```diff
@@ -91,6 +94,7 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
             mysql: ['2015-01-20 01:00:00'],
             snowflake: ['2015-01-20 01:00:00'],
             mariadb: ['2015-01-20 01:00:00.000'],
+            oracle: [new Date(Date.UTC(2015, 0, 20))],
```

Comment:
is this equivalent?

```suggestion
            oracle: [new Date(2015, 0, 20)],
```

Reply:
The time can vary based on timezone so we are considering the UTC convention